redwoods-insurance-salesforce/force-app/main/default/lwc/incidentMap/incidentMap.html

```diff
@@ -5,9 +5,9 @@ <h3 slot="title">
                 Incident Location
             </h3>
             <div class="slds-var-m-around_medium">
-                <h1 class="slds-text-heading_small">
+                <h2 class="slds-text-heading_small">
```

Comment:
@albarivas Out of curiosity, why the change from h1 to h2?

Reply:
This was one of the issues that sa11y found. Apparently heading levels should only increase by one to be accessible.

Reply:
Today I learned...